src/tests/kdbtest.c

```diff
@@ -244,7 +244,7 @@ static void
 sim_preauth(krb5_timestamp authtime, krb5_boolean ok, krb5_db_entry **entp)
 {
     /* Both back ends ignore the request parameter for now. */
-    krb5_db_audit_as_req(ctx, NULL, *entp, *entp, authtime,
+    krb5_db_audit_as_req(ctx, NULL, NULL, *entp, *entp, authtime,
```

Comment:
The comment above this line explains why it's okay to pass a null pointer for the request; it should be adjusted to also mention from.